src/components/Icon/index.tsx

```diff
@@ -0,0 +1,355 @@
+import React from 'react';
+import { Platform, StyleProp, ViewStyle } from 'react-native';
+import Image from 'react-native-fast-image';
+import { SvgProps } from 'react-native-svg';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import { Icon as IconI, IconProps as IconPropsI } from 'react-native-vector-icons/Icon';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+
+type IconConfig =
+  | ({
+    component: typeof IconI;
+  } & IconPropsI)
+  | {
+    component: typeof Image;
+    source: Function;
+  }
+  | {
+    component: React.FC<SvgProps>;
+  };
+
+const iconMap: {
+  [index: string]: IconConfig;
+} = {
+  bouy: {
```

Comment:
Let's remove all of these that came from triforce and just leave the one used in the toast